pages/en/contrib/Governance.md

```diff
@@ -20,18 +20,22 @@ Each repository has one or more lead maintainers responsible for:
 *   Seeking consensus on technical decisions.
 *   Making the final decisions when consensus cannot be achieved.
 
-Maintainers have npm publishing rights for modules and the final say on releasing new versions.
+Maintainers have npm publishing rights for modules and the final say on releasing new versions.  
```

Comment:
Hmm, I think this is not true. AFAIK, we don't give npm publishing rights to maintainers from the community. I am proposing to change "Maintainers" to "Lead maintainers" in this sentence to avoid possible confusion.

Reply:
You're right.  
How to define/find out who are the "lead maintainers"? I'm thinking, as @jjtang1 suggested, whether it's better to have "core maintainers" (i.e. the core LB team) and "community maintainers"?